packages/@aws-cdk/aws-appmesh/lib/http-route-path-match.ts

```diff
@@ -0,0 +1,85 @@
+import { CfnRoute } from './appmesh.generated';
+
+// keep this import separate from other imports to reduce chance for merge conflicts with v2-main
+// eslint-disable-next-line no-duplicate-imports, import/order
+import { Construct } from '@aws-cdk/core';
+
+/**
+ * Configuration for HTTP route match
+ */
+export interface HttpRouteMatchConfig {
+  /**
+   * Route CFN configuration for HTTP route match.
+   */
+  readonly requestMatch: CfnRoute.HttpRouteMatchProperty;
+}
+
+/**
+ * Defines HTTP route path or prefix request match.
+ */
+export abstract class HttpRoutePathMatch {
+  /**
+   * The value of the path must match the specified value exactly.
+   *
+   * @param path The exact path to match on
+   */
+  public static matchingExactly(path: string): HttpRoutePathMatch {
```

Comment:
```suggestion
  public static exactly(path: string): HttpRoutePathMatch {
```

(The word "match" already appears in the class name)

Reply:
Retaining the method name while removing `match` from the class name.